netci.groovy

```diff
@@ -257,18 +257,35 @@ if (!branch.endsWith('-ci')) {
 CreateStyleCheckTasks('./jenkins/check_eol.sh', 'ubuntu_check_eol', 'EOL Check')
 CreateStyleCheckTasks('./jenkins/check_copyright.sh', 'ubuntu_check_copyright', 'Copyright Check')
 
+// --------------
+// XPLAT BRANCHES
+// --------------
+
+def isXPlatCompatibleBranch = branch.startsWith('master') || branch.startsWith('linux') || branch.startsWith('xplat') ||
+    (branch.startsWith('release') && !(branch =~ /^release\/1\.[12](\D.*)?$/))
+
+// Include these explicitly-named branches
+def isXPlatDailyBranch = branch in ['master', 'linux', 'xplat']
+// Include some release/* branches
+if (branch.startsWith('release')) {
+    // Allows all future release/* branches which should be considered for daily builds of XPlat configs.
+    // RegEx matches e.g. release/1.1, release/1.2, release/1.3-ci, but not e.g. release/2.0, release/1.3, or release/1.10
+    releaseExclusion = branch =~ /^release\/((1\.[12](\D.*)?)|(\d+\.\d+\D.*))$/
+    isXPlatDailyBranch |= !releaseExclusion
+}
+
 // -----------------
 // LINUX BUILD TASKS
 // -----------------
 
-if (branch.startsWith('linux') || branch.startsWith('master')) {
+if (isXPlatCompatibleBranch) {
```

Comment:
I was thinking about suffixes to those branches so matching seemed like a reasonable approach, but since we only configure `*` and `*-ci` for Jenkins CI builds anyway, I think it's fine to enumerate the incompatible branches explicitly. 

I'll make the change.